openhcl/underhill_attestation/src/lib.rs

```diff
@@ -352,9 +366,13 @@ pub async fn initialize_platform_security(
         tcb_version,
     )
     .await
-    .map_err(ErrorInner::GetDerivedKeys)?;
+    .map_err(|e| {
+        tracing::error!(vmgs_encrypted = vmgs_encrypted, "failed to derive keys");
```

Comment:
Can we log `vmgs_encrypted` at the info level before this call instead?

Reply:
Updated.